scripts/jenkins/Jenkinsfile

```diff
@@ -256,6 +260,22 @@ def dockerInit() {
       "./scripts/docker/homepage/backend/Dockerfile",
       false
     )
+
+    /* Image building elektrad */
+    DOCKER_IMAGES.webui_elektrad = createDockerImageDesc(
+      "elektra/elektrad-demo", this.&idHomepage,
+      ".",
+      "./scripts/docker/web/elektrad-demo/Dockerfile",
```

Comment:
Please add some more mountpoints.

Reply:
did we really add a web directory? super confusing as homepage is not part of web. can you rename to webui?

Reply:
@markus2330 I added some new mountpoints:

```
RUN kdb mount /etc/networks system/networks hosts
RUN kdb mount /etc/ssh/ssh_config system/ssh line
RUN kdb mount /etc/ldap/ldap.conf system/ldap line
RUN kdb mount /var/lib/dpkg/available system/dpkg/available dpkg
RUN kdb mount /var/lib/dpkg/status system/dpkg/status dpkg
```

is this enough or do you have any ideas for other mountpoints?

Reply:
@ingwinlu I renamed the folder to `webui`